src/app/CommandSender.cpp

```diff
@@ -71,6 +71,10 @@ CHIP_ERROR CommandSender::SendCommandRequest(const SessionHandle & session, Syst
     VerifyOrReturnError(mpExchangeCtx != nullptr, CHIP_ERROR_NO_MEMORY);
 
     mpExchangeCtx->SetResponseTimeout(timeout);
+    if (timeout != kImMessageTimeout)
+    {
+        ChipLogError(Controller, "CommandSender: SendCommandRequest with non-default timeout");
```

Comment:
ChipLogError seems a bit strong here?

Reply:
Oh, sorry - yeah, these are going away. This was just me attempting to make sure I was hitting the code path. Everything in that tracking commit was just for the draft.